modules/tasks/tasks/src/main/resources/webapp/messages/messages.properties

```diff
@@ -53,6 +53,7 @@ task.settings.submit=Save
 
 task.history.filter.progress=Progress
 task.history.filter.tasks=Tasks
+task.history.filter.onlyLastExecution=Show only last task''s execution
```

Comment:
For properly parsing apostrophe on the UI, it is necessary to type one after the other in the messages.properties